README.md

```diff
@@ -1261,7 +1244,7 @@ const replicate = new Replicate({ useFileOutput: false });
 
 | method               | returns   | description                                                  |
 | -------------------- | ------    | ------------------------------------------------------------ |
-| `url()`              | string    | A `URL` object representing the HTTP URL or data-uri         |
```

Comment:
Small thing, but maybe let's put `url()` after `blob()` in this list so it's not the first thing people read? Ditto with the examples above. Let's put the URL example at the bottom, perhaps with the same caveat?